src/mapper/CodedValueMapper.js

```diff
@@ -3,7 +3,7 @@ import ValueMapperStore from '../helpers/ValueMapperStore';
 export class CodedValueMapper {
 
   getValue(control, value) {
-    return value && (value.displayString || value.shortName);
+    return value && (value.displayString || value.name);
```

Comment:
Add details of what is the issue this is fixing

Reply:
Since shortName is optional in openmrs concepts, the value is undefined. It should refer to fully-qualified-name i.e. 
**value.name**, this is how it is in other mappers. Eg: [BooleanValueMapper](https://github.com/Bahmni/form-controls/blob/3b751dbbb44a657854f08702b4c81eec28f7a84a/src/mapper/BooleanValueMapper.js#L6)

Reply:
Same details are updated in card also.